<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KoPageLayoutColumns</class>
  <widget class="QWidget" name="KoPageLayoutColumnsBase" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>361</width>
        <height>169</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <item row="1" column="1" >
        <widget class="QFrame" name="columnSpacingPane" >
          <property name="frameShape" >
            <enum>NoFrame</enum>
          </property>
          <property name="frameShadow" >
            <enum>Raised</enum>
          </property>
        </widget>
      </item>
      <item row="0" column="0" >
        <widget class="QLabel" name="textLabel1" >
          <property name="text" >
            <string>Columns:</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>nColumns</cstring>
          </property>
        </widget>
      </item>
      <item row="1" column="0" >
        <widget class="QLabel" name="labelSpacing" >
          <property name="text" >
            <string>Column spacing:</string>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="2" column="0" colspan="2" >
        <spacer name="spacer1" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item rowspan="3" row="0" column="2" colspan="1" >
        <widget class="QFrame" name="previewPane" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>7</vsizetype>
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape" >
            <enum>NoFrame</enum>
          </property>
          <property name="frameShadow" >
            <enum>Raised</enum>
          </property>
        </widget>
      </item>
      <item row="0" column="1" >
        <widget class="QSpinBox" name="nColumns" >
          <property name="maximum" >
            <number>16</number>
          </property>
          <property name="minimum" >
            <number>1</number>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
